src/app/containers/FrontPageMain/index.jsx

```diff
@@ -48,17 +80,22 @@ const FrontPageMain = ({ frontPageData }) => {
         <VisuallyHiddenText id="content" tabIndex="-1" as="h1">
           {offScreenText}
         </VisuallyHiddenText>
-        <Grid>
-          <GridItemConstrainedLargeWithTopMargin>
+        <FrontPageGrid columns={mainColumns} enableGelGutters>
```

Comment:
Can we remove the `main` and do the the following instead?
```
    <FrontPageGrid
        forwardedAs="main"
        role="main"
        columns={mainColumns}
        enableGelGutters
      >
        <Grid
          item
          columns={itemColumns}
          startOffset={startOffsets}
          margins={itemMargins}
        >
          <VisuallyHiddenText id="content" tabIndex="-1" as="h1">
            {offScreenText}
          </VisuallyHiddenText>
```

Reply:
> Can we remove the `main` and do the the following instead?
> 
> ```
>     <FrontPageGrid
>         forwardedAs="main"
>         role="main"
>         columns={mainColumns}
>         enableGelGutters
>       >
>         <Grid
>           item
>           columns={itemColumns}
>           startOffset={startOffsets}
>           margins={itemMargins}
>         >
>           <VisuallyHiddenText id="content" tabIndex="-1" as="h1">
>             {offScreenText}
>           </VisuallyHiddenText>
> ```

This implementation breaks the accessibility flow since the `VisuallyHiddenText` is meant to be a direct child of `main`, in this PR, the focus is to apply `psammed-grid` with the minimal restructuring of the original page.